eloquentjs/9_3_numbers_again.js

```diff
@@ -0,0 +1,16 @@
+let number = /^([+]|[-])?([.]\d+?|\d+?[.]?\d*)(e([+]|[-])?\d+?)?$/i;
```

Comment:
It would be better to call this something like `numberRegex` since calling it just `number` is slightly misleading.